core/Cargo.toml

```
@@ -184,9 +179,8 @@ minitrace = { version = "0.4.0", optional = true }
 moka = { version = "0.10", optional = true, features = ["future"] }
 once_cell = "1"
 openssh = { version = "0.9.9", optional = true }
-openssh-sftp-client = { version = "0.12.2", optional = true }
+openssh-sftp-client = { version = "0.13.3", optional = true, features = ["openssh"] }
```

I recommend enabling tracing for better log in ci and production.

```suggestion
openssh-sftp-client = { version = "0.13.3", optional = true, features = ["openssh", "tracing"] }
```